{"uuid": "caf41a54-4c7b-4334-84fc-1c77fd627a1a", "children": ["191fb1bd-70e8-4f41-be7b-6dd67eeb2a99", "36c1e201-f84f-4eb0-89c0-ed83971a849c", "966353c5-fbd9-42de-aee4-3657ec49f729", "f84a33eb-df83-415e-a1f3-c5bb07d85d76"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/136.0.7103/chromedriver_win32.zip\n", "trace": "  File \"C:\\Users\\windw\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\windw\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1122, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\windw\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 917, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\windw\\PycharmProjects\\Diplom\\Diplom_3\\tests\\conftest.py\", line 25, in driver\n    service = ChromeService(executable_path=ChromeDriverManager().install())\n                                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\windw\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\webdriver_manager\\chrome.py\", line 39, in install\n    driver_path = self._get_driver_path(self.driver)\n  File \"C:\\Users\\windw\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\webdriver_manager\\core\\manager.py\", line 30, in _get_driver_path\n    file = self._download_manager.download_file(driver.get_url())\n  File \"C:\\Users\\windw\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\webdriver_manager\\core\\download_manager.py\", line 28, in download_file\n    response = self._http_client.get(url)\n  File \"C:\\Users\\windw\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\webdriver_manager\\core\\http.py\", line 33, in get\n    self.validate_response(resp)\n    ~~~~~~~~~~~~~~~~~~~~~~^^^^^^\n  File \"C:\\Users\\windw\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\webdriver_manager\\core\\http.py\", line 16, in validate_response\n    raise ValueError(f\"There is no such driver by url {resp.url}\")\n"}, "start": 1747161715092, "stop": 1747161720409}], "start": 1747161715092, "stop": 1747161720589}